packages/button/components/ButtonBase.tsx

```diff
@@ -0,0 +1,164 @@
+import * as React from "react";
+import { cx } from "emotion";
+import {
+  buttonReset,
+  textWeight,
+  flex,
+  flexItem,
+  display,
+  padding
+} from "../../shared/styles/styleUtils";
+import {
+  button,
+  buttonBase,
+  buttonContent,
+  buttonInverse,
+  fullWidthButton,
+  getInverseMutedButtonStyles,
+  getMutedButtonStyles,
+  processingText
```

Comment:
maybe we want to import some of these classes with a different name like `processingText as processingTextStyle` it gets confusing to parse that processingText is a style class and not a variable containing processing text.

Reply:
You're right, that's more understandable